master/docs/developer/database.rst

```diff
@@ -550,6 +550,16 @@ logs
         It should only be called for finished logs.
         This method may take some time to complete.
 
+    .. py:method:: deleteOldLogChunks(older_than_timestamp)
+
+        :param integer older_than_timestamp: the logs whose step's ``started_at`` is older than ``older_than_timestamp`` will be deleted.
```

Comment:
Should it be steps or builds? Seems like it could be confusing if the first part of a build has its logs deleted but the second half doesn.t